vm/src/vm.rs

```diff
@@ -234,8 +234,10 @@ impl VirtualMachine {
                 #[cfg(not(target_arch = "wasm32"))]
                 import::import_builtin(self, "signal").expect("Couldn't initialize signal module");
 
-                import::init_importlib(self, initialize_parameter)
-                    .expect("Initialize importlib fail");
+                self.expect_pyresult(
```

Comment:
This should never happen so I am not sure if we need to print the reason to the user. What would he do in that case?

Reply:
It helped when I was breaking stuff and debugging; it's moreso printing a message to the developer.

Reply:
Maybe this should be active only in verbose mode?

Reply:
Ooh, or maybe I could check for the `RUST_BACKTRACE` environment variable and only print out the Python backtrace there? I'm not sure about the cfg, because I usually test in release mode because it's so much faster.